cip-0000/cip-0000.md

```diff
@@ -0,0 +1,393 @@
+This draft must be written in CIP style as described below, and named with an alias such as "CIP-johndoe-A-Better-Global Synchronizer" until an editor has assigned it a CIP number (authors MUST NOT self-assign CIP numbers).
+
+CIP authors are responsible for collecting community feedback on both the initial idea and the CIP before submitting it for review. However, wherever possible, long, open-ended discussions on public mailing lists should be avoided. Strategies to keep the discussions efficient include: setting up a separate special-interest mailing list for the topic, having the CIP author accept private comments in the early design phases, setting up a wiki page or git repository, etc. CIP authors should use their discretion here.
+
+It is highly recommended that a single CIP contain a single key proposal or new idea. The more focused the CIP, the more successful it tends to be. If in doubt, split your CIP into several well-focused ones.
+
+When the CIP draft is complete, a CIP editor will assign the CIP a number, label it as Standards Track, Governance, Tokenomics, Process or Informational, update the CIP Index, and merge the pull request to the cips git repository.
+The CIP editors will not unreasonably reject a CIP.
+Reasons for rejecting CIPs include duplication of effort, disregard for formatting rules, being too unfocused or too broad, being technically unsound, not providing proper motivation or addressing backwards compatibility, or not in keeping with Global Synchronizer policies or philosophy as previously defined.
+For a CIP to be accepted it must meet certain minimum criteria.
+It must be a clear and complete description of the proposed change.
+The change must represent a net improvement.
+The proposed implementation, if applicable, must be solid and must not complicate the code unduly.
+
+The CIP author may update the draft as necessary in the cips git repository. Updates to drafts should also be submitted by the author as pull requests.
+
+### Transferring CIP Ownership
+
+It occasionally becomes necessary to transfer ownership of CIPs to a new champion. In general, we'd like to retain the original author as a co-author of the transferred CIP, but that's really up to the original author. A good reason to transfer ownership is because the original author no longer has the time or interest in updating it or following through with the CIP process, or is unreachable or not responding to email. A bad reason to transfer ownership is because you don't agree with the direction of the CIP. We try to build consensus around a CIP, but if that's not possible, you can always submit a competing CIP.
+
+If you are interested in assuming ownership of a CIP, send a message asking to take over, addressed to both the original author and the CIP editors. If the original author doesn't respond to email in a timely manner, the CIP editors will make a unilateral decision (it's not like such decisions can't be reversed :).
+
+### CIP Editors
```

Comment:
@waynecollier-da based on this list @hythloda is not an editor, based on your comment in https://github.com/global-synchronizer-foundation/cips/pull/20#discussion_r1958430995 she is, sounds like we may need to update the list here.